addons/core/addon/helpers/format-day-year.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+import Helper from '@ember/component/helper';
+import { inject as service } from '@ember/service';
+
+/**
+ * Takes a `Day` instance and returns in year format if there are no remainders,
+ *  * @example
+ * {{format-day-year number}}
+ *
+ **/
+export default class extends Helper {
+  // =services
+  @service intl;
+
+  // =compute method
+
+  /**
+   * @param {number} days
+   * @return string
+   */
+
+  translation(year) {
+    if (year === 1) {
+      return this.intl.t('titles.year');
+    } else {
+      return this.intl.t('titles.years');
+    }
+  }
+  compute([days]) {
+    const remainder = days % 365;
+    if (remainder === 0) {
+      const numberOfYears = Math.floor(days / 365);
+      return numberOfYears + ' ' + this.translation(numberOfYears);
+    } else {
+      return days + ' ' + this.intl.t('titles.days');
```

Comment:
Shouldn't this and the years be in the translation itself still? We also then don't need to do the `if (year === 1) `check since the `intl` service handles [pluralization](https://formatjs.io/docs/core-concepts/icu-syntax/#plural-format)

Reply:
Do you mean to pass the translation from the template? ty, I'll look into pluralization

Reply:
I just meant passing in `days` or `numberOfYears` to `intl.t` so it can handle the 0, 1, or other case rather than us doing if else checks